pandas-iris/{{ cookiecutter.repo_name }}/src/tests/test_pipeline.py

```diff
@@ -0,0 +1,9 @@
+"""
+This is a boilerplate test file for pipeline 'data_science'
```

Comment:
Don't forget to update now the `data_science` pipeline isn't there anymore.

Reply:
Would it be better just to remove these headers entirely?

Reply:
Yeah actually I think that's fine.

Reply:
Although for this test file I would keep it otherwise there's nothing in it.